blocks.js

```diff
@@ -60,17 +60,10 @@ module.exports = function (file, block_size, cache) {
         bufs.push(block.slice(read_start, read_end))
         start += (read_end - read_start)
 
-        if(start < end) next(i+1)
-        else {
+        if (start < end) {
+          next(i+1)
+        } else {
           var buffer = bufs.length == 1 ? bufs[0] : Buffer.concat(bufs)
-          if(!buffer.length)
```

Comment:
why did you remove this? doesn't your code just return a buffer of zeros? when you read this?

Reply:
Thanks, nice catch. I thought that this was throwing errors on buffers of nullbytes, but it looks like this still passes tests so I'm not sure where I got that idea. I've re-added it now.